src/fill-zulipdb-with-msgs-deploy.mjs

```diff
@@ -0,0 +1,217 @@
+// Reference to TypeScript definitions for IntelliSense in VSCode
+/// <reference path="../rnode-grpc-gen/js/rnode-grpc-js.d.ts" />
+
+import grpcLib from '@grpc/grpc-js';
+import util from 'util';
+
+// RNode with environment parameters
+import { rnodeService } from './rnode-env.mjs';
+
+// Load .env file
+import { config } from 'dotenv';
+config();
+
+import postgres from 'postgres';
+
+/**
+  * @param {object} arg
+  * @param {typeof path} arg.path
+  */
+async function main(env, {grpcLib}) {
+  const zulip_db_config = {
+    host: 'localhost',
+    port: 5432,
+    database: 'zulip',
+    username: 'zulip',
+    password: process.env.POSTGRES_PASSWORD,
+  };
+  
+  const { DB_CONTRACT_URI } = env;
+
+  // Postgres connection
+  const sql = postgres(zulip_db_config);
+
+  // FIXME this function is returning just zerver_message and zerver_usermessage
+  function getTables(){
+    return `
+      new return,
+        lookup(\`rho:registry:lookup\`)
+      in {
+        new dbCh, tablesCh in {
+          lookup!(\`${DB_CONTRACT_URI}\`, *dbCh) |
+          for (db <- dbCh) {
+            db!("tables", *tablesCh) |
+            for (@tables <- tablesCh) {
+              return!(("Tables", tables))
+            }
+          }
+        }
+      }
+    `;
+  } 
+
+  function getTableKeys(tableName){
+    return `
+      new return,
+        lookup(\`rho:registry:lookup\`)
+      in {
+        new dbCh, keysCh in {
+          lookup!(\`${DB_CONTRACT_URI}\`, *dbCh) |
+          for (db <- dbCh) {
+            db!("keys", "${tableName}", *keysCh) |
+            for (@keys <- keysCh) {
+              return!(("keys", keys))
+            }
+          }
+        }
+      }
+    `;
+  }
+
+  function getRecordInDB(table, ids){
+    return `
+      new return,
+        lookup(\`rho:registry:lookup\`)
+      in {
+        new dbCh, selectCh in {
+          lookup!(\`${DB_CONTRACT_URI}\`, *dbCh) |
+          for (db <- dbCh) {
+            db!("select", "${table}", [${ids}], *selectCh) |
+            for (@keys <- selectCh) {
+              return!(("select", keys))
+            }
+          }
+        }
+      }
+    `;
+  }
+
+  //TODO does exist a library that could make me avoid this messy code to extract data?
+  function getTablesNames(result) {
+    const tables = [];
+    const exprsList = result.postblockdataList[0]?.exprsList;
+    if (exprsList){
+      const objList = exprsList[0]?.eTupleBody.psList;
+      for (let obj of objList) {
+        let body = obj.exprsList[0]?.eSetBody;
+        if (body) {
+          for (let list of body.psList) {
+            let setBody = list.exprsList[0]?.gString;
+            tables.push(setBody);
+          }
+        }
+      }
+    }
+    return tables;
+  }  
```

Comment:
After our conversation about returning AST object, I've added helper function [rhoParToJson](https://tgrospic.github.io/rnode-grpc-js/modules.html#rhoParToJson) as part of gRPC API generator. It should help to avoid this manual work with conversion to JSON.

Reply:
done